roles/openshift_adm/README.md

```diff
@@ -27,9 +27,11 @@ This role requires the following parameters to be configured.
 * `cifmw_openshift_adm_dry_run` (bool) If enabled, no modifications are
   performed on the cluster.
 * `cifmw_openshift_adm_login_retry_count` (int) The maximum number of attempts
-  to be made for a OpenShift login success. Default is `100`.
+  to be made for a OpenShift login success. Default is `10`.
```

Comment:
Could you please explain this change in the commit message?

Reply:
Thank you. 

I have amended the commit message to reflect the reason for the change.